data/json/recipes/other/medical.json

```diff
@@ -32,9 +32,9 @@
     ],
     "tools": [ [ [ "surface_heat", 20, "LIST" ] ] ],
     "components": [
-      [ [ "rag", 3 ] ],
-      [ [ "filament", 300, "LIST" ], [ "cordage", 1, "LIST" ], [ "superglue", 1 ] ],
-      [ [ "hard_liquor_toxic", 14, "LIST" ] ]
+      [ [ "rag", 3 ], [ "felt_patch", 3 ], [ "leather", 3 ], [ "fur", 3 ] ],
```

Comment:
I doubt bandages made out of fur, leather, or wool felt are going to be terribly useful.

Reply:
Imported change from More Survival Tools. Given the historical use of wet rawhide for this purpose (or for torture, wrapping a limb in wet rawhide is dangerous if you don't know how much it'll shrink), this seems reasonable.

Reply:
Please provide sources for these materials being used as bandages if you're going to make this claim.
If it requires extensive special handling, such as, "apply wet with just the right amount of tightness", it's not comparable and you need to reverse the change.

Reply:
Also need a source for raw garlic being an effective antiseptic.

Reply:
Note that I said rawhide, not leather. Leather is generally treated specifically so it won't expand like that when wet. Either way, this is a makeshift first aid kit, is it not? in fact it's not accurate to call it a makeshift first aid kit as its properties are more akin to a bandage. In which case why are we excluding other usable clean materials to use as a dressing?

As for garlic, again the apparent purpose of the antiseptic item in this recipe is sterilizing a dressing, not topical use. If you wish however:

https://academic.oup.com/jac/article/43/6/837/754741

https://www.ncbi.nlm.nih.gov/pubmed/10594976

Both concern consumption of garlic rather than topical use, but the indicated results have even more interesting implications in my opinion. Another source covers it briefly, but I wouldn't know with 100% certainly how accurate their advice is: https://www.survivalright.com/prevent-infection.html

Reply:
In other words, there's no indication that these would work, please revert those changes.

Reply:
...did you actually read the first two links? Because they kinda stated "antibacterial effect" rather overtly. What they were testing is whether it could actually have an *antibiotic* effect which is even more of a stretch than using allicin as a disinfectant.

Plus, why was this accepted for More Survival Tools but not mainline? But alright, removing garlic from the components list.

Reply:
>why was this accepted for More Survival Tools but not mainline?

Mods can do what they please. The only important requirement is that they should be working and don't bring bugs along with them.

Reply:
It would be quite nonsensical for a mod that seems based on "can I do something in real life that would make the game more enjoyable in a wilderness playthrough" to toss reality out the window.

Reply:
>to toss reality out the window.

It might be not tossing reality out, but conscientious delusion on some matter.

Reply:
Given every sources I can find maintains that garlic can indeed kill bacteria, I'd say if it's inaccurate it wasn't a failure on the mod author's part. Honestly I'd take "I literally CAN'T find anyone debunking what garlic does" to be a consistent yes, but it's Kevin's call.

Reply:
Closest I've been able to find to actually debunking this, to any extent, would be from here: https://www.aafp.org/afp/2005/0701/p103.html#afp20050701p103-b2

"In 1858, Pasteur noted garlic’s antibacterial activity, and it was used as an antiseptic to prevent gangrene during World War I and World War II."

"Small studies have shown that garlic exerts antimicrobial activity against gram-positive and gram-negative bacteria, viruses, fungi, and parasites.2 Topical and dietary garlic traditionally has been used in the treatment of infections—particularly digestive, respiratory, and dermatologic infections—ranging from diarrhea and vaginitis to colds and warts, but few good clinical studies support this use."

Unfortunately the site's sources are in the form of bibliography without links, and both sections site a text with the rather dubious title of "The healing power of herbs: the enlightened person’s guide to the wonders of medicinal plants"

Reply:
> Given every sources I can find maintains that garlic can indeed kill bacteria,

You've posted a single extremely sketchy webpage that claims that garlic, along with lemon juice, toothpaste and mouthwash make for good topical antiseptics, several sources that by your own admission say nothing about topical use and one source that clearly states a lack of evidence.

> but few good clinical studies support this use.

Please drop it.

Also in case it was lost, drop fur, leather and felt.